kani-driver/src/harness_runner.rs

```diff
@@ -39,12 +44,8 @@ impl<'sess, 'pr> HarnessRunner<'sess, 'pr> {
         self.check_stubbing(harnesses)?;
 
         let sorted_harnesses = crate::metadata::sort_harnesses_by_loc(harnesses);
-
         let pool = {
-            let mut builder = rayon::ThreadPoolBuilder::new();
-            if let Some(x) = self.sess.args.jobs() {
-                builder = builder.num_threads(x);
-            }
+            let builder = rayon::ThreadPoolBuilder::new();
```

Comment:
```suggestion
            let mut builder = rayon::ThreadPoolBuilder::new();
            if let Some(x) = self.sess.args.jobs() {
                builder = builder.num_threads(x);
            }
```

Is there a reason why you reverted this? If not, can you please revert it?